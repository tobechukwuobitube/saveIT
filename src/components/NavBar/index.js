import React from "react";
import { Link } from 'react-router-dom';
Link

export default function NavBar({onSignout}) {
  return (
    <div>
      <div className="navigation">
        <Link to="/">
          <p className="logo">Banka</p>
        </Link>
        <nav id="nav">
          <ul>
            <li>
              <Link className="btn" to="/">
                Home
              </Link>
            </li>
            <li>
              <Link className="btn" to="/login" onClick={onSignout}>
                Logout
              </Link>
            </li>
          </ul>
        </nav>
      </div>
    </div>
  );
}
