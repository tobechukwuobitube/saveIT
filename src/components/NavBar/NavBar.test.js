import React from "react";
import NavBar from './index';

test("NavBar snapshot test", () => {
  const wrapper = shallow(<NavBar />);
  expect(wrapper).toMatchSnapshot();
});
