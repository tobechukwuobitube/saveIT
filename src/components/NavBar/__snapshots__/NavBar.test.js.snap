// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar snapshot test 1`] = `ShallowWrapper {}`;
